README.md

```diff
@@ -67,16 +66,49 @@ Add the following (multi-)task to your `Gruntfile`:
       all: {
         src: [ 'some/valid.json' ],
         options: {
+          // parsing
+          mode: 'json',
+          ignoreComments: false,
+          ignoreTrailingCommas: false,
+          allowSingleQuotedStrings: false,
+          allowDuplicateObjectKeys: true,
+          // formatting
           format: true,
+          prettyPrint: false,
           indent: 2,
-          sortKeys: false
+          sortKeys: false,
+          pruneComments: false,
+          stripObjectKeys: false
         }
       }
     }
 
+<<<<<<< HEAD
```

Comment:
Merge markers - incomplete merge?

Reply:
Oops, sorry, I must've missed it. Git could detect it itself...